.github/workflows/test-os.yml

```diff
@@ -20,7 +20,7 @@ on:
 
 env:
   GO_VERSION: "1.21"
-  TESTFLAGS: "-v --parallel=6 --timeout=30m"
+  TESTFLAGS: "-v --parallel=6 --timeout=60m"
```

Comment:
not for this PR but if this gets too long we need to split this suite to multiple parallel workers

Reply:
Sure, for now we just have those 3.